.github/workflows/ghcr_push.yml

```diff
@@ -39,6 +39,8 @@ jobs:
             path: integrations/flytekit_plugins
           - name: pandera
             path: integrations/flytekit_plugins
+          - name: greatexpectations
```

Comment:
There's one `great_expectations` directory within the parent `greatexpectations` which houses the configuration. Hence, I don't think we can change this. Please look at the directory structure: https://github.com/flyteorg/flytesnacks/blob/f4a6e36440ca1fb51f5203858ee10ae2069aa5d2/cookbook/integrations/flytekit_plugins/greatexpectations/great_expectations/expectations/sqlite/movies.json

Reply:
Hi @samhita-alla, I thought I would check if this could be changed, since it looks like a few of the other directories in this list have underscores. I understand if this is not possible.

Reply:
Yeah. We can have `great_expectations`, but since there's this config directory `great_expectations` present within it, it may confuse the users. The internal config directory is the one that's directly generated from Great Expectations' CLI.

Reply:
I see - that makes sense. Let me bring this up with the team at our meeting next week to see if anyone has any ideas.

Reply:
Sure!

Reply:
Just following up - I brought this up with the team, and the consensus was that if it is possible, we would like to use `great_expectations/great_expectations`. If you think it will be too confusing, we can leave as is. Thanks!

Reply:
`greatexpectations` is now `great_expectations`.

Reply:
I've reverted this to `greatexpectations` because there's a conflict between `great_expectations` in this directory and the usual `great_expectations` library in the plugin code.